src/components/TodosProvider/TodosProvider.tsx

```diff
@@ -0,0 +1,163 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { useLocalStorage } from '../../utils/LocalStorage';
+
+const sTodos: Todo[] = [
+  {
+    id: 1,
+    title: '1 todo',
+    completed: false,
+  },
+  {
+    id: 2,
+    title: '2 todo',
+    completed: false,
+  },
+  {
+    id: 3,
+    title: '3 todo',
+    completed: true,
+  },
+  {
+    id: 4,
+    title: '4 todo',
+    completed: false,
+  },
+];
+
+export const TodoContext = React.createContext(sTodos);
+
+export const TodoUpdateContext = React.createContext({
+  addTodo: (_title: string) => {},
+  deleteTodo: (_todoId: number) => {},
+  editTodo: (_todo: Todo) => {},
+  completeTodo: (_todoId: number) => {},
+  uncompleteTodo: (_todoId: number) => {},
+  removeAllCompleted: () => {},
+  toogleAll: (_completed: boolean) => {},
```

Comment:
why you need **_** before name arguments?

Reply:
It is a hint for IDE to say, that i don't use this variable inside function.